### PR TITLE
feat(@vtmn/css): add skeleton component

### PR DESCRIPTION
**Pull request by lauthieb**
## Pull Request checklist

🚨 Please review the [guidelines for contributing](../blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] Check PR name, it must follow the https://www.conventionalcommits.org pattern
- [x] If it's a new component in CSS, ask for design review

## Description

Add skeleton component in CSS

## Does this introduce a breaking change?

- No

❤️ Thank you!

### CODE REVIEW

File: packages/sources/css/src/components/skeleton/src/index.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  background-image: linear-gradient(
+    90deg,
+    rgba(255, 255, 255, 0) 0,
+    rgba(255, 255, 255, 0.3) 10%,
+    rgba(255, 255, 255, 0.6) 40%,
+    rgba(255, 255, 255, 0.6) 60%,
+    rgba(255, 255, 255, 0.3) 90%,
+    rgba(255, 255, 255, 0)
+  );
+  animation: shimmer 1.5s infinite;
```
Comment (GaspardMathon) [inline packages/sources/css/src/components/skeleton/src/index.css:31]: I would add a ease-in-out property to make it more dynamic

File: packages/sources/css/src/components/skeleton/src/index.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+}
+.vtmn-skeleton_line {
+  height: 8px;
+  border-radius: 8px;
+}
+.vtmn-skeleton_avatar {
+  width: 32px;
+  height: 32px;
+  border-radius: 50%;
+}
```
Comment (GaspardMathon) [inline packages/sources/css/src/components/skeleton/src/index.css:42]: Should we not use rem() function for sizing element ?

**Review by GaspardMathon — COMMENTED** [review]
Some detailed changes

File: packages/sources/css/src/components/skeleton/src/index.css (hunk 1 above)
Comment (lauthieb) [reply packages/sources/css/src/components/skeleton/src/index.css]: You're right, done!

File: packages/sources/css/src/components/skeleton/src/index.css (hunk 2 above)
Comment (lauthieb) [reply packages/sources/css/src/components/skeleton/src/index.css]: You're right, done!

**Review by GaspardMathon — APPROVED** [review]

**Pull request merged**